content/en/continuous_integration/explorer/_index.md

```diff
@@ -90,7 +90,7 @@ Test run data is available in [dashboards][102] and [notebooks][103], enabling b
 {{% /tab %}}
 {{% tab "Pipeline Executions" %}}
 
-Navigate to [**CI** > **Pipeline Executions**][101] to see your CI pipeline execution results across the following levels: **Pipeline**, **Stage**, **Job**, **Step**, and **Command**.
+Navigate to [**Software Delivery** > **CI Visibility** > **Executions**][101] to see your CI pipeline execution results across the following levels: **Pipeline**, **Stage**, **Job**, **Step**, and **Command**.
```

Comment:
It looks like this page is still called Pipeline Executions in the UI. Is there a reason for dropping `Pipeline` from the description here?

Reply:
Answered comment above!